tools/harness-thci/OpenThread.py

```diff
@@ -3123,23 +3124,26 @@ def __detectZephyr(self):
 
     def __discoverDeviceCapability(self):
         """Discover device capability according to version"""
-        if self.IsBorderRouter:
+        thver = self.__executeCommand('thread version')[0]
+        if thver in ['1.3', '4'] and not self.IsBorderRouter:
+            self.log("Setting capability of {}: (DevCapb.C_BR13 | DevCapb.C_Host13 | DevCapb.C_Comm)".format(self))
```

Comment:
```suggestion
            self.log("Setting capability of {}: (DevCapb.C_BR13 | DevCapb.C_Host13)".format(self))
```
Is the Commissioner needed for Thread 1.3 reference device?

Reply:
@edmont My understanding from @karthick-grl and Nithin at GRL was that we needed this as written above, but we can discuss with them on Thursday if you don't believe this should be there.  I believe this is what the test harness is currently expecting.

Reply:
I can only find `DevCapb.C_Comm` being used in C_5.10.13 and C_5.10.14 tests, not for any 1.3 tests.

Reply:
I'll remove it then.  I think they were just trying to future proof this.